LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
Is there anything more recent than 2004 that we should be using?

Reply:
I took a further look, but I don't think so.  Apache 2.0 is currently a popular choice for open software.  Its main competitors for relatively simple and relatively open licenses seem to be the MIT license and the GPLv3.0 license, neither of which appear to protect against re-use of the Rubin logo.  In fact, in FAQ for a GPVv3.0 (which itself is from 2007), it is written:
_"What is the difference between the Apache License 2.0 and the GNU GPL? The GNU GPL is a copyleft license. So software that uses any GPL-licensed component has to release its full source code and all rights to modify and distribute the entire code. The Apache License 2.0 doesn't impose any such terms."_